cattle.go

```diff
@@ -25,13 +25,13 @@ func NewCattleClient(cattleUrl string, accessKey string, secretKey string) (*Cat
 	}, nil
 }
 
-func (c *CattleClient) UpdateServiceDomainName(serviceDnsRecord utils.ServiceDnsRecord) error {
+func (c *CattleClient) UpdateServiceDomainName(metadataRecord utils.MetadataDnsRecord) error {
 	event := &rancher.ExternalDnsEvent{
 		EventType:   "dns.update",
-		ExternalId:  serviceDnsRecord.Fqdn,
-		ServiceName: serviceDnsRecord.ServiceName,
-		StackName:   serviceDnsRecord.StackName,
-		Fqdn:        serviceDnsRecord.Fqdn,
+		ExternalId:  metadataRecord.DnsRecord.Fqdn,
+		ServiceName: metadataRecord.ServiceName,
+		StackName:   metadataRecord.StackName,
+		Fqdn:        utils.UnFqdn(metadataRecord.DnsRecord.Fqdn),
```

Comment:
@janeczku I wanted to know what are the possible custom FQDN templates one can use currently via our catalog? (I guess the customizations are limited to the  %{{service_name}}, %{{stack_name}}, %{{environment_name}} placeholders only, but just the separators can be custom? )